justfile

```diff
@@ -0,0 +1,43 @@
+# Define variables
+set dotenv-required
+
+DB_CONTAINER_NAME := "mina-mesh-db"
+
+# Download archive DB dump
+dl network:
+    ./tasks/dl.sh --network {{network}}
+
+# Initialize PostgreSQL database
+pg-init:
+    docker run -d --name {{DB_CONTAINER_NAME}} -p 5432:5432 -v $(pwd)/sql_scripts:/docker-entrypoint-initdb.d -e POSTGRES_PASSWORD=$POSTGRES_PASSWORD -e POSTGRES_USER=$POSTGRES_USER postgres postgres -c log_statement=all -c log_min_duration_statement=0
```

Comment:
When doing `pg-init` I got:

```
sh: 1: POSTGRES_PASSWORD: parameter not set
error: Recipe `pg-init` failed on line 12 with exit code 2
```

Perhaps we can add in justfile:
```
POSTGRES_USER := "mina"
POSTGRES_PASSWORD := "whatever"
```

or define it in the `.env.example`?